src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -17805,7 +17805,7 @@ instruct vsra8B_imm(vecD dst, vecD src, immI shift) %{
 
```

Comment:
match rule for vsra8B_imm should also be changed.

Reply:
Changed